tests/product/base_product_case.py

```diff
@@ -241,17 +248,8 @@ def check_if_corrupted_rpm(self):
                            self.presto_rpm_filename))
 
     def copy_presto_rpm_to_master(self):
-        if not self.presto_rpm_filename:
-            # TODO: once the RPM is on Maven Central, pull the RPM from there
-            self.presto_rpm_filename = 'presto-0.101-1.0.x86_64.rpm'
-            rpm_path = os.path.join(prestoadmin.main_dir,
-                                    self.presto_rpm_filename)
-            urllib.urlretrieve('http://teradata-download.s3.amazonaws.com/'
-                               'aster/presto/lib/presto-0.101-1.0.x86_64.rpm',
-                               rpm_path)
-        else:
-            rpm_path = os.path.join(prestoadmin.main_dir,
-                                    self.presto_rpm_filename)
+        rpm_path = os.path.join(prestoadmin.main_dir,
```

Comment:
u already created the rpm_path in the method above, so this might not be needed?